tfrobot/pkg/deployer/deployer.go

```diff
@@ -333,13 +333,13 @@ func getBlockedNodes(groupDeployments groupDeploymentsInfo) []uint64 {
 	return blockedNodes
 }
 
-func buildDeployment(vmGroup Vms, nodeID uint32, networkName, vmName, solutionType, sshKey string) workloads.Deployment {
+func buildDeployment(vmGroup Vms, nodeID uint32, networkName, vmName, solutionType, sshKey string, isLight bool) workloads.Deployment {
 	disks, diskMounts := parseDisks(vmName, vmGroup.SSDDisks)
 	volumes, volumeMounts := parseVolumes(vmName, vmGroup.Volumes)
 
 	deployment := workloads.NewDeployment("", nodeID, solutionType, nil, networkName, disks, nil, nil, nil, nil, volumes)
 
-	if !vmGroup.WireGuard && !vmGroup.PublicIP4 && !vmGroup.PublicIP6 && !vmGroup.Ygg {
+	if isLight {
```

Comment:
I think here we missed public ips check

Reply:
it's already tested in the filtration [here](https://github.com/threefoldtech/tfgrid-sdk-go/pull/1247/files#diff-23ae2427f94dd747eb10339c3d6297e7310b5ae3de2792e7b747c50ef693a22dR59)